.github/workflows/pypi_upload.yml

```diff
@@ -0,0 +1,58 @@
+name: Upload Python Package
+
+on:
+  workflow_call:
+    inputs:
+      whl-file-name:
+        required: true
+        type: string
+      tar-file-name:
+        required: true
+        type: string
```

Comment:
Is this something you manually enter when triggering the build?

Reply:
The `workflow_call` event allows workflows to be called from other workflows, so these inputs are specified there.

If this had a `workflow_dispatch` event, that would allow manually triggering the workflow, in which case inputs could be manually entered - although in the places where I have done that, it has been to specify a URL where the input asset can be downloaded from.